### ISSUE TITLE
FTP access

### ISSUE THREAD

**Issue by thany** [meganz/sdk#2256]
Not sure if this is the right place, but it the closest one, it seems.

I would like to be able to connect to my MEGA drive through FTP (SFTP preferably) *without* having to use MEGAcmd.

After all, MEGAcmd is not available on Android, and Android is exactly where uploading is extrmely troublesome, both on the website and through the app. FTP access would be a great way for any FTP-capable app (like Total Commander) to gain access.

**Comment by thany** [comment]
@diegocr This has nothing to do with the client, mind you. What I;m asking for, is *direct* FTP access, i.e. strictly not through some locally running client.

**Comment by diegocr** [comment]
Yeah, sorry, but keep in mind we cannot setup S/FTP to run through the web, if we ever build an Electron app that could be possible using the webclient codebase, although 1) it won't work for your use-case, and 2) it wouldn't perform better than an sdk-powered client.

So, i think your best bet is having such a client running in your desktop machine or a NAS, and then Android communicating with it. 

SDK guys should be able to provide better advice than me in this regard here.
